src/main/scala/com/typesafe/sbt/packager/linux/RpmNoReplaceConfigPlugin.scala

```diff
@@ -0,0 +1,18 @@
+package com.typesafe.sbt.packager.linux
+
+import com.typesafe.sbt.packager.archetypes.JavaServerAppPackaging
+import com.typesafe.sbt.packager.rpm.RpmPlugin
+import com.typesafe.sbt.packager.rpm.RpmPlugin.autoImport
+import sbt._
+
+/**
+  * This plugin automatically updates all config files in a RPM to "noreplace". Those files will not be overwritten
+  * during an update if they have been changed on disk. This is useful for server apps.
+  */
+object RpmNoReplaceConfigPlugin extends AutoPlugin with LinuxKeys with LinuxMappingDSL {
```

Comment:
I'm still not convinced of my initial idea implementing this as an autoplugin. It only makes sense when enabled by default and for debian as well. 

This will break backwards compatibility, which I usually prefer to avoid. If you can make a good case ( maybe some best practices, blog posts on this issue? ) I'm fine with adding this.

From a code perspective:

- instead of extending the `LinuxKeys` use the "global" [Keys](https://github.com/sbt/sbt-native-packager/blob/master/src/main/scala/com/typesafe/sbt/packager/Keys.scala) object